### PR TITLE
CIF-690 - Refactor Magento's customer-deployment configuration files

### PR DESCRIPTION
**Pull request by cjelger**
- splitted configuration parameters in 2 files
- moved all configuration parameters to serverless.yml file

### CODE REVIEW

**Comment by cjelger** [comment]
NOT READY TO BE MERGED --> we will have to fix the `circleCI` configuration if/once we agree about the configuration split. Right now the ITs will fail.

File: customer-deployment/serverless.yml [hunk 1]
```diff
@@ -11,9 +11,12 @@ provider:
   ignore_certs: true
 
 custom:
-  customer-namespace: ${opt:customer-namespace, file(package.json):customer-namespace}
-  customer-package: ${opt:customer-package, file(package.json):customer-package}
-  bindings-namespace: ${opt:bindings-namespace, file(package.json):bindings-namespace}
+  # Adobe I/O Runtime configuration. The variables can be overriden by command-line arguments.
+  customer-namespace: ${opt:customer-namespace, 'cjelger'}
```
Comment (mhaack) [inline customer-deployment/serverless.yml]: I think we should move these to a separate config file as well or use placeholder here. But not store personal namespace info here.

Comment (cjelger) [reply customer-deployment/serverless.yml]: Sure my mistake, this should be replaced by a placeholder. BTW, *you* asked me to remove that from `package.json` so I don't think we should ask an extra config file, but we can also ad those in `credentials.json`.

Comment (mhaack) [reply customer-deployment/serverless.yml]: `credentials.json` is fine

**Pull request merged**